src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Metadata/Models/ContentTypeExtensions.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using OrchardCore.ContentManagement.Metadata.Settings;
+
+namespace OrchardCore.ContentManagement.Metadata.Models;
+
+public static class ContentTypeExtensions
+{
+    public static bool HasStereotype(this ContentTypeDefinition contentTypeDefinition)
+    {
+        return !String.IsNullOrEmpty(contentTypeDefinition.GetSettings().Stereotype);
+    }
+
+    public static string GetStereotype(this ContentTypeDefinition contentTypeDefinition)
+    {
+        return contentTypeDefinition.GetSettings().Stereotype;
+    }
+
+    public static bool IsListable(this ContentTypeDefinition contentTypeDefinition)
```

Comment:
I don't think these change should belong to this PR

Reply:
I think these extensions are related in a way. They should clean up some of the code used in may places. Additionally, some of these methods are used in this PR. I recommend we keep them and start using them in other RPs

Reply:
I see